include/psa/crypto_sizes.h

```diff
@@ -208,6 +208,18 @@
  * operations, and does not need to accept all key sizes up to the limit. */
 #define PSA_VENDOR_RSA_MAX_KEY_BITS 4096
 
+/* The minimum size of an RSA key on this implementation, in bits.
+ * This is a vendor-specific macro.
+ *
+ * Limits RSA key generation to a minimum due to security reasons.
```

Comment:
Not really security reasons: it's to reduce the risk of accidentally creating a key size that's smaller than intended due to a bits/bytes confusion.

Reply:
changed it to " to avoid accidental misuse."